SimPEG/DataMisfit.py

```diff
@@ -1,65 +1,49 @@
 from __future__ import print_function
 import numpy as np
-from SimPEG import Utils
-from SimPEG import Survey
+import properties
 
+from . import Utils
+from . import Survey
+from . import ObjectiveFunction
+from .Tests import checkDerivative
 
-class BaseDataMisfit(object):
-    """BaseDataMisfit
+
+class BaseDataMisfit(ObjectiveFunction.L2ObjectiveFunction):
+    """
+    BaseDataMisfit
 
         .. note::
 
-            You should inherit from this class to create your own data misfit term.
+            You should inherit from this class to create your own data misfit
+            term.
     """
 
     debug   = False  #: Print debugging information
-    counter = None   #: Set this to a SimPEG.Utils.Counter() if you want to count things
+    counter = None  #: Set this to a SimPEG.Utils.Counter() if you want to count things
```

Comment:
@rowanc1: I am not actually sure what this does. I left it there as it was in the original Regularization.